SECURITY.md

```diff
@@ -0,0 +1,58 @@
+# Security Policy
+The Royal Navy Standards security policy is to avoid leaving the ecosystem worse than we found it. Meaning we are not planning to introduce vulnerabilities into the ecosystem.
+
+The Royal Navy Standards team takes security bugs in the standards-toolkit seriously.  We appreciate your efforts to responsibly disclose your
+findings, and will make every effort to acknowledge your contributions.
+
+## Supported Versions
+
+The below table details which versions of the standards-toolkit are supported with bug fixes and security updates:
+
+| Version | Supported          |
```

Comment:
Can this be simplified so that `1.x` is not supported and `2.x` is? Or something similar...